src/pages/TrueSightV2/pages/RegisterWhitelist/SubscribeForm.tsx

```diff
@@ -57,8 +57,8 @@ export default function EmailForm({
   const checkReferCodeExist = useCallback(
     async (code: string) => {
       try {
-        if (!code) return
-        const { data } = await checkReferalCode(code)
+        if (!code.trim()) return

```

Comment:
do we need to check `code` nullable? 🤔 
```suggestion
        if (!code?.trim()) return

```